changelogs/fragments/add_diff_mode_for_state_file.yaml

```
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - Added diff mode for .tfstate file (https://github.com/ansible-collections/cloud.terraform/pull/11).
```

Technically correct, but not as useful to users of the collection. I suggest

```suggestion
  - Support diff mode in cloud.terraform.terraform both in check mode and in non-check mode. (https://github.com/ansible-collections/cloud.terraform/pull/11).
```

---

Updated as suggested.